app/src/main/java/vn/mbm/phimp/me/PhimpMe.java

```diff
@@ -63,7 +67,8 @@
 //        forceCloseDialogAfterToast = false,
 //        resToastText = R.string.crash_report_text)
 @SuppressWarnings("deprecation")
-public class PhimpMe extends AppCompatActivity implements BottomNavigationView.OnNavigationItemSelectedListener //, android.view.GestureDetector.OnGestureListener
+public class PhimpMe extends AppCompatActivity implements BottomNavigationView.OnNavigationItemSelectedListener,
+        ViewPager.OnPageChangeListener //, android.view.GestureDetector.OnGestureListener
```

Comment:
@pa1pal I rebased the code and it completely got messed up. 
Shall I close this and open a new PR

please neglect my latest commits to this.